src/tests/unit/platform/test_users.py

```diff
@@ -269,6 +255,27 @@ def test_get_grant(cbcsdk_mock):
     assert grant.roles == ["psc:role::SECOPS_ROLE_MANAGER", "psc:role:test:APP_SERVICE_ROLE"]
 
 
+def test_delete_user_without_grant(cbcsdk_mock):
+    """Tests deleting a user that does not have a grant."""
+    cbcsdk_mock.mock_request('GET', '/appservices/v6/orgs/test/users', GET_USERS_RESP)
+    cbcsdk_mock.mock_request('POST', '/access/v2/grants/_fetch', wrap_grant(None))
+    cbcsdk_mock.mock_request('DELETE', '/appservices/v6/orgs/test/users/3911', None)
+    api = cbcsdk_mock.api
+    user = api.select(User).user_ids([3911]).one()
```

Comment:
This should use our better practice like so
```
user = api.select(User, 3911)
```

Reply:
Actually, I'm wrong, the code has a contingency for that by calling _refresh(), which knows how to fake it. I'll change this.

Reply:
Perfect!